content/events/kick-off.yaml

```diff
@@ -19,30 +19,30 @@
     url: https://techlabsorg.typeform.com/to/bEsJr5Gh
```

Comment:
We will create new typeforms for every event, so I would recommend to also comment them out.